{
  "contest": 226,
  "handle": "0xbrett8571",
  "risk": "3",
  "title": "An attacker can call back into the `withdraw()` function before it completes leading to reentrancy attacks.",
  "issueId": 393,
  "issueUrl": "https://github.com/code-423n4/2023-03-asymmetry-findings/issues/393"
}